docs/js/home.js

```diff
@@ -129,14 +129,15 @@ $(function () {
   function checkMenuOverWhite() {
     var $navbar = $('.site-header .navbar');
     var $whiteBottom = $('.white-bottom');
-    var currentIsMenuOverWhite = overlap($navbar, $whiteBottom);
+    var $header = $('header');
+    var currentIsMenuOverWhite = divOver($navbar, $whiteBottom);
 
     if (currentIsMenuOverWhite !== isMenuOverWhite) {
       isMenuOverWhite = currentIsMenuOverWhite;
       if (isMenuOverWhite) {
-        TweenMax.to('header.site-header nav.navbar a', 0.1, {color:'rgba(0,0,0,1)'});
+        $header.addClass('over-white-background');
       } else {
-        TweenMax.to('header.site-header nav.navbar a', 0.1, {color:'rgba(255,255,255,1)'});
+        $header.removeClass('over-white-background');
```

Comment:
(this code was removed now), since it was a callback in a timer (interval), then I thought toggle could mess things up, dunno. Maybe not indeed